packages/visual-group/src/group-helper/matrix-model.js

```diff
@@ -248,6 +262,9 @@ export const getMatrixModel = (dataModel, facetsAndProjections, valueCellCreator
         });
     }
 
+    formatKeys(columnKeys, colFacets.map(facetField => facetField.rawFormat()));
```

Comment:
colFacets.map(facetField => facetField.rawFormat()) can be used in the function as well